### PR TITLE
SoftwareArtifact as range for assessedElement

### PR DESCRIPTION
**Pull request by ilans**
See: https://github.com/spdx/spdx-3-model/issues/911

### CODE REVIEW

**Comment by bact** [comment]
- [  ] To update CHANGE LOG
- [  ] To update diagrams

**Review by goneall — APPROVED** [review]
LGTM - Thanks @ilans

**Comment by zvr** [comment]
I've just realized that this introduces a dependency between the Security profile and the Software one. I don't know whether we had it before. Are we OK with this ?
@goneall @ilans @kestewart

**Comment by zvr** [comment]
So, I've just checked. The dependency wasn't there before.
Can we simply make it `/Core/Artifact` to spare us long(er) discussions and more possible changes?

File: model/Security/Classes/VulnAssessmentRelationship.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ assessment relationships. It factors out the common properties shared by them.
 ## Properties
 
 - assessedElement
-  - type: /Core/Element
+  - type: /Software/SoftwareArtifact
```
Comment (goneall) [inline model/Security/Classes/VulnAssessmentRelationship.md:23]:
```suggestion
  - type: /Core/Artifact
```

File: model/Security/Properties/assessedElement.md [hunk 2]
```diff
@@ -16,4 +16,4 @@ to specify the precise location where a vulnerability was found.
 
 - name: assessedElement
 - Nature: ObjectProperty
-- Range: /Core/Element
+- Range: /Software/SoftwareArtifact
```
Comment (goneall) [inline model/Security/Properties/assessedElement.md:19]:
```suggestion
- Range: /Core/Artifact
```

**Review by goneall — CHANGES_REQUESTED** [review]
Per Alexios' suggestion, added suggestion changing to `/Core/Artifact`

File: model/Security/Classes/VulnAssessmentRelationship.md (hunk 1 above)
Comment (goneall) [reply model/Security/Classes/VulnAssessmentRelationship.md:23]: From call on 5 Dec - agreed to leave as SoftwareArtifact

File: model/Security/Properties/assessedElement.md (hunk 2 above)
Comment (goneall) [reply model/Security/Properties/assessedElement.md:19]: From call on 5 Dec - agreed to leave as SoftwareArtifact

**Review by goneall — APPROVED** [review]

**Review by bact — APPROVED** [review]

**Pull request merged**